app/js/lib/reducers/form_content.js

```diff
@@ -5,9 +5,31 @@ var objectAssign = require( 'object-assign' ),
 	initialState = {
 		amount: 0,
 		isCustomAmount: false,
-		paymentType: 'BEZ'
+		paymentType: 'BEZ',
+		recurringPayment: 0, // 0 = one-off, 1 = recurring Payment
```

Comment:
These are fields right? Then can't you have an isRecurring method? I don't like the `isFalsy` stuff below. Defensive coding like this obfuscates intend to some degree.